index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse Bootcamp YouTube PlayList</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/bd260a0fea.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <!-- navigation-bar  -->
+    <section>
+        <nav class="top-nav">
+            <ul class="top-left">
+                <li>
+                    <button type="submit"><i class="fa-solid fa-bars"></i></button>
+                </li>
+                <li class="my-tube">
+                    <button type="submit" class="yt-icon"><i class="fa-brands fa-youtube"></i></button>   
+                    <button type="submit" class="yt-logo">YouTube</button>
+                </li>
+            </ul>
+            <ul class="center">
+                <li>
+                    <form class="form" action="https://www.google.com/search" >
+                        <input class="type-text" type="text" placeholder="Search Here!" name="g">
+                        <button type="submit"><i class="fa fa-magnifying-glass"></i></button>
+                    </form>
+                </li>
+                <li>
+                    <button type="submit" class="mic-icon"><i class="fa-solid fa-microphone"></i></button>
+                </li>
+            </ul>
+            <ul class="top-right">
+                <li>
+                    <button type="submit" class="top-right-icons"><i class="fa-solid fa-video"></i></button>
+                </li>
+                <li>
+                    <button type="submit" class="top-right-icons"><i class="fa-solid fa-video"></i></button>
+                </li>
+                <li>
+                    <button type="submit" class="top-right-icons"><i class="fa-solid fa-video"></i></button>
+                </li>
+                <li>
+                    <button type="submit" class="top-right-icons"><i class="fa-solid fa-video"></i></button>
+                </li>
+                <li>
+                    <button type="submit" class="top-right-icons"><i class="fa-solid fa-video"></i></button>
+                </li>
+            </ul>
+        </nav>
+    </section>
+    <!-- nav-link-end  -->
+    <section>
+        <div  class="flex-container">
+            <ul class="left-bar-video" >
+                <li>
+                    <iframe width="180" height="110" src="https://www.youtube.com/embed/l5yNJWKnoDg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+                </li>
+            </ul>
```

Comment:
Kindly add a short description to the playlist that will better help our users to understand what content of the video it is. 🤓